boards/arm/stm32l562e_dk/pinmux.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2020 Linaro Limited
```

Comment:
Please update

Reply:
Thanks for the review. The copyright comment has been updated.